### DISCUSSION TITLE
Waves V14 unable to scan

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#1202] [Help]
Hi,
I just updated on my mac Waves from V13 to V14. 
Since it always ends with an error : unable to scan Waveshell V14.x and AG server starts./
V13 were scanned correctly with a previous beta.

I wonder if this is not an issue with the timeout changes during the scan : the error appears not long ago after the scan of everything else is finished. Sometimes it may last a very long time to scan all waves plugins (I remember clicking dozens of time on the popup timeout when it existed in AG).

Can timeout be modified in the settings ?
Any idea how to solve that ?  I cant use any waves plugin.

Thanks

**Comment by Henri75** [comment]
Happens with both VST3 and VST on Mac M1 (native)
I tried several times to rescan all the VST3 waveshell but it always fails

**Reply by apohl79** [reply]
Could you provide some scan logs? I don't have access to waves V14 shells.
